sdk/communication/Azure.Communication.CallAutomation/src/Models/CreateCallOptions.cs

```diff
@@ -55,5 +55,10 @@ public CreateCallOptions(CallSource callSource, IEnumerable<CommunicationIdentif
         /// Repeatability Headers.
         /// </summary>
         public RepeatabilityHeaders RepeatabilityHeaders { get; set; }
+
+        /// <summary>
+        /// The identifier of the Cognitive Service resource assigned to this call.
```

Comment:
On service side, I think it's been updated:
/// The endpoint URL of the Azure Cognitive Services resource attached

https://skype.visualstudio.com/SCC/_git/sync_calling_concore-platform/pullrequest/720937?_a=files&path=/Microsoft.Skype.Platform/Microsoft.Skype.Platform.ExecutionAgent/Azure.Communication.Service.ServerCalling/Contracts/V4-2023-01-15-preview/Requests/AnswerCallRequest.cs

Reply:
will correct the comment

Reply:
will correct the comment